http/CredentialsManager.cc

```diff
@@ -125,19 +112,8 @@ CredentialsManager::~CredentialsManager() {
     for (auto &item: creds) {
         delete item.second;
     }
-    creds.clear();
```

Comment:
Why remove this call to clear? The previous for loop deletes each AccessCredentials object  in the map {{creds}}, but it does not change the contents of cress. Thus crews is left full of broken pointers.